ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -94,7 +94,10 @@ if [[ -n "${TEST_SKIPS}" ]]; then
     TEST_ARGS="${TEST_ARGS:-} --file /tmp/tests"
 fi
 
-openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" ${TEST_ARGS:-} \
+# using set -x so we will see the full openshift-tests command in the log. disable right after.
+set -x
+openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" "${TEST_ARGS:-}" \
     --provider "${TEST_PROVIDER}" \
     -o /tmp/artifacts/e2e.log \
     --junit-dir /tmp/artifacts/junit
+set +x
```

Comment:
do we need to turn it off after?  That's the end of the script.

Reply:
was just trying to keep xtrace off in the case that some future development happens at the end of this script.

Reply:
there's no reason to ever put additional stuff at the end of this script.  If anything needs to run post-test, it should go in a separate step.

Reply:
ack